### PR TITLE
docs: fixed and updated twitter logo

### PR DESCRIPTION
**Pull request by Shreyas0410**
## Description

In the current Readme the logo of twitter is not visible and is not updated to X.

Screenshot 2023-11-01 033406

This Fixes this issue 

Screenshot 2023-11-01 033453

## Checklist

- [X] I’ve added tests to confirm my change works.
- [X] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [X] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [X] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by Shreyas0410** [comment]
@fisker @sosukesuzuki can you please check.

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@
   <a href="#badge">
     <img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square"></a>
   <a href="https://twitter.com/PrettierCode">
-    <img alt="Follow Prettier on Twitter" src="https://img.shields.io/twitter/follow/prettiercode.svg?label=follow+prettier&style=flat-square"></a>
+    <img alt="Follow Prettier on Twitter" src="https://img.shields.io/twitter/follow/prettiercode.svg"></a>
```
Comment (fisker) [inline README.md:58]:
```suggestion
    <img alt="Follow Prettier on Twitter" src="https://img.shields.io/badge/follow-@PrettierCode-9f9f9f.svg?style=flat-square"></a>
```

WDYT?

Comment (fisker) [reply README.md:58]: Or

```
https://img.shields.io/badge/Follow-%40PrettierCode-9f9f9f?style=flat-square&logo=x
```

Comment (fisker) [reply README.md:58]: Or just the logo

```
https://img.shields.io/badge/-%40PrettierCode-9f9f9f?style=flat-square&logo=x&labelColor=000
```

Comment (sosukesuzuki) [reply README.md:58]: I prefer just the logo pattern

**Pull request merged**